TEDU-103 review of how to update Mautic (#90)
mautic/user-documentation: docs/getting_started/how_to_update_mautic.rst

If your instance is in production, has a large number of Contacts and/or is  on shared hosting, it's **strongly** recommended that you update at the command line.

.. warning::

>     Updating in the User interface requires a significant amount of resources, and can be error-prone if the server restricts resource allocation. A failed update or corrupted data can result from this. This feature would be completely removed in Mautic 5.0 and you have to update at the command line.

Reviewer: ```suggestion
    Updating in the User interface requires a significant amount of resources, and can be error-prone if the server restricts resource allocation. A failed update or corrupted data can result from this. It's planned to remove this feature in Mautic 5.0, requiring updating at the command line.
```

The update feature within the Mautic User interface (UI) has been deprecated from Mautic 4.2, but you will be alerted within the UI (see below figure) when a new version of the Mautic is available. 

.. image:: images/gui-update-deprecated.png
  :width: 700
  :height: 200
  :alt: Screenshot showing deprecated update feature warning

.. warning::
    Before starting to upgrade, it's highly recommended to take a backup of your instance. If updates are available, you will receive an update notification followed by step-by-step instructions in the command-line interface to complete the process.

Log in via the command line, and change directory to the Mautic directory using the command:

> ``cd /your/mautic/directory``

Reviewer: Should we use the proper code-block here? Or is there value in just using the inline code markup?

Author: inline code markup works
